TheAmazingAudioEngine/Utilities/AEIOAudioUnit.m

```diff
@@ -31,6 +31,7 @@
 #import "AETime.h"
 #import "AEAudioBufferListUtilities.h"
 @import AVFoundation;
+#import <mach/mach_time.h>
```

Comment:
I've actually wrapped all the mach_time functions in AETime.h, so no need for that